### ISSUE TITLE
How to get more knowledge about Openfarm users, gardeners and contributors?

### ISSUE THREAD

**Issue by sophiakc** [openfarmcc/OpenFarm#939]
We need more information about what:
- gardeners need and are looking for when landing on openfarm.cc
- Open Source contributors can bring to the table

Right now, anyone can enter the Slack channel. We don't know who they are, whether they are gardeners willing to ask questions about their garden, or if they are contributors and which skills and for how many min/ hours per week they can bring.
On the other side, Openfarm needs:
- help on implementing features, fixing bugs, improving usability of the website — right now there is not enough people to do so
- more gardeners creating and using guides, so far it's mainly dev contributors maintaining the database, we need real gardeners!

So far, the Slack channel has been used to gather the contributors, it has been done in an unstructured way. Out of 300 members, there is maybe 5 actually active. And unless, we ask one by one, which is not sustainable, we have no idea about what are the skills and availabilities of people.

Since we don't have any metrics of any kind, regarding our users, whether gardeners or contributors, I've suggested to implement a form to serve this purpose. The form is the entry point of the slack channel, which enables people to access it and understand how it works.
The Slack is already a tech-savvy obstacle for most people, including gardeners willing to ask questions about gardening. And the Slack is dominated by code related topics.

I'm open to other alternatives than the form to get more knowledge about the users. 
Let's discuss about that here:

**Comment by simonv3** [comment]
Note: I'm fairly sure the form's already implemented and in use on the website. 

I still want to give people an option to skip it if they don't want to fill it in.

**Comment by roryaronson** [comment]
I think the form is a good place to start. Let's give that a go for a little bit and then can maybe re-write some of the questions/inputs, or change direction if the form isn't working well.

**Comment by roselynemakena** [comment]
Just to weigh in abit on this issue, I feel like the form is quite fundamental in collecting user information and is indeed one of the best ways to do this in this case just as @sophiakc has mentioned..However, having it by-passable should highly be considered in this case, since not all users do want to fill it in or they are neither farmers nor contributors or they want to check out open farm before giving out any information - and we wouldn't want to ignore these users. Another option to consider in this issue would be to reduce the length of the form from 6 pages to maybe 1 or 2 pages.That way, we can collect just the most crucial information about them and collect more later.

**Comment by sophiakc** [comment]
I agree with @roryaronson that we can give this a try for a period of time and then iterate on it. 
So far, we have less than 10 answers to the form so far, which isn't enough. I'll keep checking it and see what kind of information we get out of it, now that it is implemented on the website.
Also, this set of data will be matched with the Google Analytics of Openfarm, so we'll be able to see if there are a lot of clicks on the different entry points to the form vs. the actual number of people answering it ;-)
